shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -355,12 +355,24 @@ const CodeEditor = createClass({
 				let text = '';
 				let currentLine = from.line;
 				const maxLength = 50;
+
+				let headerText = '';
+				let otherText = '';
 				while (currentLine <= to.line && text.length <= maxLength) {
-					text += this.codeMirror.getLine(currentLine);
-					if(currentLine < to.line)
-						text += ' ';
-					currentLine += 1;
+					const currentText = this.codeMirror.getLine(currentLine);
+					currentLine++;
+					if(!currentText) {
+						continue;
+					}
```

Comment:
When `!currentText` we are at the end of the loop anyway, right? Is this needed?

Reply:
Initially it was to skip out if the string was blank, I've tested it and it does not appear to be necessary.